config/checker-framework-suppressions/checker-nullness-optional-interning-suppressions.xml

```diff
@@ -6205,7 +6201,6 @@
     </details>
   </checkerFrameworkError>
 
-  <!-- The following 2 fields are initialized during parsing step -->
```

Comment:
Is there way to keep such comments?

Reply:
Comments restored. Actually I have not noticed that it has been removed